keywords/biometrics.md

```diff
@@ -1,8 +1,9 @@
 ## Biometrics
 
-Biometrics refers to the measurement and statistical analysis of people's unique physical and behavioral characteristics[^Gillis2021biometric].Biometric authentication is used to verify and identify the identity of an individual[^Das2015biometric]. This way of identification is often used in settings like the airport security, health care and law enforcement[^Gillis2021biometric]. There are two types of biometric technologies, which include physical and behavioral. The physical biometric technologies examining the unique features of our biological self, while the behavioral is related to our mannerisms[^Das2016biometric]. Examples of biometric technologies include fingerprint recognition, vein pattern recognition, hand geometry recognition, iris recognition, retinal recognition, facial recognition, keystroke recognition and signature recognition[^Das2015biometric].
+Biometrics refers to the measurement and statistical analysis of people's unique physical and behavioral characteristics[^Gillis2021biometric].Biometric authentication is used to verify and identify the identity of an individual[^Das2015biometric]. This way of identification is often used in settings like the airport security, health care and law enforcement[^Gillis2021biometric]. In the context of new media, biometrics can be user in fields such as [UX](../main/glossary.md#ux) to provide a more seamless user experience. For instance, biometric authentication is used to make it easier for user to open their phone or easier to save and store passwords and can even be found in smartwatches[^Schwartz2017].There are two types of biometric technologies, which include physical and behavioral. The physical biometric technologies examining the unique features of our biological self, while the behavioral is related to our mannerisms[^Das2016biometric]. Examples of biometric technologies include fingerprint recognition, vein pattern recognition, hand geometry recognition, iris recognition, retinal recognition, facial recognition, keystroke recognition and signature recognition[^Das2015biometric].
```

Comment:
This is good. Please note that references as footnotes always go after the punctuation marks, not before.